iib/web/static/api_v1.yaml

```diff
@@ -132,8 +136,9 @@ paths:
             application/json:
               schema:
                 oneOf:
-                  - $ref: '#/components/schemas/RequestIndexImageVerbose'
-                  - $ref: '#/components/schemas/RequestRegenerateBundleVerbose'
+                  - $ref: '#/components/schemas/AddResponseVerbose'
+                  - $ref: '#/components/schemas/RmResponseVerbose'
+                  - $ref: '#/components/schemas/RegenerateBundleResponseVerbose' #TODO do other belongs here?
```

Comment:
I don't think others belong here. It should be safe to remove the TODO

Reply:
removed